dune

```diff
@@ -647,6 +648,15 @@
   (backend/asm_targets/.asm_targets.objs/byte/asm_targets.cmt
    as
    compiler-libs/asm_targets.cmt)
+  (backend/debug/dwarf/dwarf_low/.dwarf_low.objs/byte/dwarf_low.cmi
```

Comment:
I think all of the individual `.cmi` files need to be listed, unfortunately.  Also the three lists slightly higher up in the file for the `.a`, `.cma` and `.cmxa` files need updating.  Those lists must be in topologically-sorted order as per the comment.

Reply:
Done